README.md

```diff
@@ -67,6 +67,13 @@ using another playbook (for example):
 
 `ansible-playbook -e @config/openhpc.yml -i ansible/inventory_openhpc --vault-password-file=vault-password ansible/openhpc.yml`
 
+To setup and mount hyperconverged glusterfs:
+
+```
+ansible-playbook -e @config/openhpc.yml -i ansible/inventory_openhpc --vault-password-file=vault-password ansible/gluster_cluster.yml
```

Comment:
Is it gluster-cluster not gluster_cluster?

Reply:
You're right. Although, this is now imported into openhpc playbook so no need to go at lengths in README